### DISCUSSION TITLE
Open Endpoint CSS

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/datalinq-community#12] [General]
Hallo!

Ich würde das Styling aus meiner View auslagern und hier verspeichern:
![image](https://github.com/gis-eni/datalinq-forum/assets/127403099/19ad9c9d-fb1b-456f-a70c-614d005542b6)

Leider scheine ich nicht den richtigen Pfad in meiner View eingetragen zu haben...bitte um Unterstützung wie das Styling zugegriffen werden kann. Laut DevTools liegen sie unter cssproxy...
![image](https://github.com/gis-eni/datalinq-forum/assets/127403099/0b9a742c-dd6a-4dc1-8676-de8f76c8b9de)

Danke für die Unterstützung!

**Comment by gis-eni** [comment]
Ich versteh die Frage nicht richtig.

Wenn du ``Endpoint CSS`` verwendest braucht du dich nicht um Pfade kümmern. Das CSS wird automatisch in jeden (Report) View unter diesem Endpoint geladen. 
``cssproxy`` ist nur ein Proxy, der das CSS in den Browser bringt. Der eigentliche Speicherort ist irgendwo im *WebGIS Storage*

**Reply by ue-black** [reply]
Jetzt hat es funktioniert. Ich war nur verwirrt weil manche webgis-styles meine styles überschrieben haben und deshalb nicht gezogen haben. z.B. navbar...
Danke für die rasche Antwort!
